docs/static/css/material_dropdown.css

```diff
@@ -0,0 +1,57 @@
+/* https://codepen.io/mildrenben/pen/RPwQEY */
+
+.drop {
+    width: 125px;
+    vertical-align: middle;
+}
+
+.drop button {
+    color: inherit;
+    font-weight: 700;
+    font-size: .65rem;
+    cursor: pointer;
+}
+
+.drop button:after {
+    content: "\e5c5";
+    padding: 0 0 0.3rem 0.2rem;
+    font-size: 0.5rem;
+    font-family: "Material Icons";
+}
+
+.drop ul {
+    position: absolute;
+    left: 0;
+    top: 3em;
```

Comment:
It's more beautiful or ugly? just suggest `top: 5.8em;`
 
![image](https://user-images.githubusercontent.com/61399850/151096086-5a214cb4-66ba-4777-8f51-389e8bf93a5e.png)

Reply:
In that case, that little triangle pointing down should have an animation to pointing up. I don't want to implement it.

Seems that overflow menu is also one of the recommended way to implement a menu.

![image](https://user-images.githubusercontent.com/8463288/151118493-51784980-73c1-4870-b801-b50cdff302ba.png)

https://material.io/components/menus/android#dropdown-menus

Reply:
It looks more suitable for some sidebar buttons on mobile，I think `Popup menus` is good for this. FYI